freeswitch_click2dial/freeswitch_click2dial.py

```diff
@@ -0,0 +1,416 @@
+        fs_number = self.convert_to_dial_number(erp_number)
+        # Add 'out prefix'
+        if fs_server.out_prefix:
+            _logger.debug('Out prefix = %s', fs_server.out_prefix)
+            fs_number = '%s%s' % (fs_server.out_prefix, fs_number)
+        _logger.debug('Number to be sent to FreeSWITCH = %s', fs_number)
+
+        # The user should have a CallerID
+        if not user.callerid:
+            raise UserError(_('No callerID configured for the current user.'))
+
+        variable = ""
+        if user.freeswitch_chan_type == 'user':
+            # We can only have one alert-info header in a SIP request
+            if user.alert_info:
+                variable += 'alert_info=' + user.alert_info
+            elif fs_server.alert_info:
+                variable += 'alert_info=' + fs_server.alert_info
+            if user.variable:
+                for user_variable in user.variable.split('|'):
+                    if len(variable) and len(user_variable):
+                        variable += ','
+                    variable += user_variable.strip()
+        if user.callerid:
+            caller_name = re.search(r'([^<]*).*',
+                                    user.callerid).group(1).strip()
+            caller_number = re.search(r'.*<(.*)>.*',
+                                      user.callerid).group(1).strip()
+            if caller_name:
+                if len(variable):
+                    variable += ','
+                caller_name = caller_name.replace(",", r"\,")
+                variable += 'effective_caller_id_name=\'' + caller_name + '\''
+            if caller_number:
+                if len(variable):
+                    variable += ','
+                variable += 'effective_caller_id_number=\'' + \
+                            caller_number + '\''
+            if fs_server.wait_time != 60:
+                if len(variable):
+                    variable += ','
+                variable += 'ignore_early_media=true' + ','
+                variable += 'originate_timeout=' + str(fs_server.wait_time)
+        if len(variable):
+            variable += ','
+        variable += 'odoo_connector=true'
+        channel = '%s/%s' % (user.freeswitch_chan_type, user.resource)
+        if user.dial_suffix:
+            channel += '/%s' % user.dial_suffix
+
+        try:
+            # originate <csv global vars>user/2005 1005 DP_TYPE DP_NAME
+            #    'Caller ID name showed to aleg' 90125
+            dial_string = (('<' + variable + '>') if variable else '') + \
+                channel + ' ' + fs_number + ' ' + fs_server.context + ' ' + \
+                '\'' + self.get_name_from_phone_number(fs_number) + '\' ' + \
+                fs_number
+            fs_manager.api('originate', dial_string.encode('utf-8'))
+        except Exception, e:
+            _logger.error(
+                "Error in the Originate request to FreeSWITCH server %s",
+                fs_server.ip_address)
+            _logger.error(
```

Comment:
No need to double call `_logger`, just add parametr `exc_info=True` to previous `_logger.error` call